src/main/java/com/nike/cerberus/operation/core/CreateLoadBalancerOperation.java

```diff
@@ -65,8 +63,9 @@ public void run(final CreateLoadBalancerCommand command) {
         final String environmentName = environmentMetadata.getName();
         final VpcOutputs vpcOutputs = configStore.getVpcStackOutputs();
 
-        final String sslCertificateArn = configStore.getServerCertificateArn(Stack.CMS)
-                .orElseThrow(() -> new IllegalStateException("Could not retrieve SSL certificate ARN!"));
+        // Use latest cert, if there happens to be more than one for some reason
+        final String sslCertificateArn = configStore.getCertificationInformationList()
```

Comment:
Does this work with the ALB CloudFormation? From what I understand in your previous comment this method will return the certificate name, not the ARN.

At the least we should rename this to the cert name

Reply:
fixed